src/client.ts

```diff
@@ -586,14 +589,38 @@ export class UXDClient {
     );
   }
 
-  public createSetMangoDepositoryQuoteMintAndRedeemFeeInstruction(
-    quoteFee: number,
+  public createEditMangoDepositoryInstruction(
+    controller: Controller,
+    depository: MangoDepository,
+    authority: PublicKey,
+    fields: {
+      quoteMintAndRedeemFee: number;
+    },
+    options: ConfirmOptions
+  ): TransactionInstruction {
+    return this.instruction.editMangoDepository(
+      {
+        quoteMintAndRedeemFee: fields.quoteMintAndRedeemFee,
+      },
+      {
+        accounts: {
+          authority: authority,
+          controller: controller.pda,
+          depository: depository.pda,
+        },
+        options: options,
+      }
+    );
+  }
+
+  public createDisableDepositoryRegularMintingInstruction(
+    disableMinting: boolean,
     controller: Controller,
     depository: MangoDepository,
     authority: PublicKey,
     options: ConfirmOptions
   ): TransactionInstruction {
-    return this.instruction.setMangoDepositoryQuoteMintAndRedeemFee(quoteFee, {
+    return this.instruction.disableDepositoryRegularMinting(disableMinting, {
```

Comment:
git is miss-interpreting this as a change, but it's actually just the functions being re-ordered